lib/java/polyglot-ydoc-server/src/main/java/org/enso/polyfill/Main.java

```diff
@@ -0,0 +1,82 @@
+package org.enso.polyfill;
+
+import java.io.File;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.Executors;
+
+import org.enso.polyfill.crypto.CryptoPolyfill;
+import org.enso.polyfill.encoding.EncodingPolyfill;
+import org.enso.polyfill.timers.TimersPolyfill;
+import org.enso.polyfill.web.AbortControllerPolyfill;
+import org.enso.polyfill.web.EventTargetPolyfill;
+import org.enso.polyfill.web.PerformancePolyfill;
+import org.enso.polyfill.websocket.WebSocketPolyfill;
+import org.graalvm.polyglot.Context;
+import org.graalvm.polyglot.HostAccess;
+import org.graalvm.polyglot.Source;
+import org.graalvm.polyglot.io.IOAccess;
+
+public class Main {
+
+    private static final String YDOC_SERVER_PATH = "/dist/assets/ydocServer.js";
+
+    private Main() {
+    }
+
+    public static void main(String[] args) throws Exception {
+        var ydoc = ClasspathResource.createTempFile(YDOC_SERVER_PATH);
+        var commonJsRoot = new File(ydoc).getParent();
+
+        HostAccess hostAccess = HostAccess.newBuilder(HostAccess.EXPLICIT)
+                .allowArrayAccess(true)
+                .build();
+
+        var b = Context.newBuilder("js")
+                .allowIO(IOAccess.ALL)
+                .allowHostAccess(hostAccess)
+                .allowExperimentalOptions(true)
+                .option("js.commonjs-require", "true")
```

Comment:
I think we can remove these `js.commonjs...` options. We don't need them if everything gets packed into single `.js` file by webpack or co.